packages/block-editor/src/components/block-card/index.js

```diff
@@ -49,7 +46,7 @@ function BlockCard( { title, icon, description, blockType, className } ) {
 
 	return (
 		<div className={ classnames( 'block-editor-block-card', className ) }>
-			{ isOffCanvasNavigationEditorEnabled && parentNavBlockClientId && (
+			{ parentNavBlockClientId && (
```

Comment:
It does feel strange exposing this even though I believe it will only apply if there is a parent Nav block and thus won't appear for all blocks.

I feel we ought to have some indication that this is Nav block specific code inside the block editor package. A comment maybe?

Reply:
the condition does show it's nav block specific. it's also probably going away as soon as the overall effort to make these drill down panels easily configurable works out.